tests/src/test/java/alluxio/master/JournalShutdownIntegrationTest.java

```diff
@@ -183,7 +182,7 @@ public void singleMasterJournalStopIntegration() throws Exception {
     cluster.stopFS();
     CommonUtils.sleepMs(TEST_TIME_MS);
     // Ensure the client threads are stopped.
-    mExecutorsForClient.shutdown();
+    mExecutorsForClient.shutdownNow();
```

Comment:
@calvinjia The purpose of the sleep might have been to wait for the response of a create to reach the client from the master. I think it makes sense to terminate the client later.